cypress/e2e/projects.cy.ts

```diff
@@ -0,0 +1,138 @@
+import '../support';
+import {
+    createProjectItem,
+    createSelectButton,
+    projectCancelButton,
+    projectCreateForm,
+    projectDescriptionInput,
+    projectKeyPredictor,
+    projectKeyPredictorError,
+    projectKeyPredictorHint,
+    projectKeyPredictorInput,
+    projectSubmitButton,
+    projectTitleInput,
+} from '../../src/utils/domObjects';
+import { exactUrl } from '../helpers';
+import { routes } from '../../src/hooks/router';
+import { keyPredictor } from '../../src/utils/keyPredictor';
+
+const testProjectTitle = 'Test project title in e2e';
+const testProjectDescription = 'Test project description in e2e';
+const testProjectKey = keyPredictor(testProjectTitle);
+const customKey = 'CUSTOMKEY';
+
+describe('Projects', () => {
+    beforeEach(() => {
+        cy.signInViaEmail();
+    });
+
+    describe('create', () => {
+        describe('ways to open dialog', () => {
+            // NB: no way to test in all locales, but this test runs via keyCode.
+            // It means doesn't matter what letter on key. It must work.
+            it('should open and close dialog via hotkey', () => {
+                cy.wait(50);
```

Comment:
why are you use the `wait` function?

Reply:
please leave the comment about this `wait` statement :)

Reply:
I did https://github.com/taskany-inc/issues/pull/1536/commits/b7a04d2a2f346857a0136a974dedf52d7bcf01c6#diff-a0b339d4018897995c753568420dd9538a8bb735f60fc6f79ed9b77c4b539ec8R33-R37